tests/json-fixtures/test_blockchain.py

```diff
@@ -116,7 +115,14 @@ def test_blockchain_fixtures(fixture_name, fixture):
     #     assert rlp.encode(genesis_header) == fixture['genesisRLP']
 
     db = MemoryDB()
-    evm = MainnetEVM.from_genesis(
+
+    evm = MainnetEVM
+    if fixture_name.startswith('Homestead'):
+        evm = EVM.configure(
+            'HomesteadEVM',
+            vm_configuration=[(0, HomesteadVM)])
```

Comment:
*note*

It would be great if we can figure out an API for *re-configuring* start block numbers that was more elegant.  Fully re-specifying the `vm_configuration` is not elegant.  It isn't clear what this looks like but seemed worth surfacing to get more people thinking about it.